stock_measuring_device_zippcube/models/product_packaging.py

```diff
@@ -0,0 +1,28 @@
+from odoo import api, fields, models
+
+
+class ProductPackaging(models.Model):
+    _inherit = "product.packaging"
+
+    scan_device_id = fields.Many2one(
```

Comment:
this can't have a generic name if it's tied to a specific model.
Unless you plan to have a generic model `measurement.device`